include/CLI/StringTools.hpp

```diff
@@ -13,6 +13,14 @@
 #include <vector>
 
 namespace CLI {
+
+/// output streaming for enumerations
+template <typename T, typename = typename std::enable_if<std::is_enum<T>::value>::type>
+std::ostream &operator<<(std::ostream &in, const T &level) {
```

Comment:
Nice addition that I didn't think of. It might be a good idea to make it opt-in in some way though, so a user doesn't clash if they already have something similar.

Reply:
Fantastic, inside an exported namespace it works great, I'm adding this to my PR if you don't mind...

Reply:
Feel free,  It took a little while to figure out where to actually put this in the code so it was actually visible to all the functions that might need it.  I pulled it from the enum.cpp example, so it would work with the transformer Validator.